engine/src/test/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActorSpec.scala

```diff
@@ -241,39 +241,6 @@ class MaterializeWorkflowDescriptorActorSpec extends CromwellTestKitWordSpec wit
       system.stop(materializeWfActor)
     }
 
-
-    "reject a taskless WDL source" ignore {
```

Comment:
Not entirely related to the rest of the PR, but this test case is just invalid now